src/code/z_parameter.c

```diff
@@ -65,125 +65,22 @@ Input sPostmanTimerInput[4];
-    { SCENE_31MISAKI, RESTRICTIONS_NONE },
-    { SCENE_TAKARAKUJI, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 3, 0, 0, 1) },
-    { SCENE_UNSET_3A, RESTRICTIONS_NONE },
-    { SCENE_TORIDE, RESTRICTIONS_NONE },
-    { SCENE_FISHERMAN, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 3, 0, 0, 1) },
-    { SCENE_GORONSHOP, RESTRICTIONS_INDOORS },
-    { SCENE_DEKU_KING, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0) },
-    { SCENE_LAST_GORON, RESTRICTIONS_MOON },
-    { SCENE_24KEMONOMITI, RESTRICTIONS_NONE },
-    { SCENE_F01_B, RESTRICTIONS_NO_DOUBLE_TIME },
-    { SCENE_F01C, RESTRICTIONS_NO_DOUBLE_TIME },
-    { SCENE_BOTI, RESTRICTIONS_NONE },
-    { SCENE_HAKUGIN_BS, RESTRICTIONS_NONE },
-    { SCENE_20SICHITAI, RESTRICTIONS_NONE },
-    { SCENE_21MITURINMAE, RESTRICTIONS_NONE },
-    { SCENE_LAST_ZORA, RESTRICTIONS_MOON },
-    { SCENE_11GORONNOSATO2, RESTRICTIONS_NONE },
-    { SCENE_SEA, RESTRICTIONS_NONE },
-    { SCENE_35TAKI, RESTRICTIONS_NONE },
-    { SCENE_REDEAD, RESTRICTIONS_NONE },
-    { SCENE_BANDROOM, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0) },
-    { SCENE_11GORONNOSATO, RESTRICTIONS_NONE },
-    { SCENE_GORON_HAKA, RESTRICTIONS_NONE },
-    { SCENE_SECOM, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 3, 0, 3, 0, 0, 0) },
-    { SCENE_10YUKIYAMANOMURA, RESTRICTIONS_NONE },
-    { SCENE_TOUGITES, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 3, 3, 3, 3, 0, 0) },
-    { SCENE_DANPEI, RESTRICTIONS_NO_DOUBLE_TIME },
-    { SCENE_IKANAMAE, RESTRICTIONS_NONE },
-    { SCENE_DOUJOU, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 3, 0, 0, 1) },
-    { SCENE_MUSICHOUSE, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0) },
-    { SCENE_IKNINSIDE, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 3, 3, 3, 0, 0, 0) },
-    { SCENE_MAP_SHOP, RESTRICTIONS_INDOORS },
-    { SCENE_F40, RESTRICTIONS_NONE },
-    { SCENE_F41, RESTRICTIONS_NONE },
-    { SCENE_10YUKIYAMANOMURA2, RESTRICTIONS_NONE },
-    { SCENE_14YUKIDAMANOMITI, RESTRICTIONS_NONE },
-    { SCENE_12HAKUGINMAE, RESTRICTIONS_NONE },
-    { SCENE_17SETUGEN, RESTRICTIONS_NONE },
-    { SCENE_17SETUGEN2, RESTRICTIONS_NONE },
-    { SCENE_SEA_BS, RESTRICTIONS_NONE },
-    { SCENE_RANDOM, RESTRICTIONS_NONE },
-    { SCENE_YADOYA, RESTRICTIONS_INDOORS },
-    { SCENE_KONPEKI_ENT, RESTRICTIONS_NONE },
-    { SCENE_INSIDETOWER, RESTRICTIONS_SET(0, 0, 0, 0, 3, 3, 3, 3, 3, 0, 0, 0) },
-    { SCENE_26SARUNOMORI, RESTRICTIONS_NO_DOUBLE_TIME },
-    { SCENE_LOST_WOODS, RESTRICTIONS_NONE },
-    { SCENE_LAST_LINK, RESTRICTIONS_MOON },
-    { SCENE_SOUGEN, RESTRICTIONS_MOON },
-    { SCENE_BOMYA, RESTRICTIONS_INDOORS },
-    { SCENE_KYOJINNOMA, RESTRICTIONS_NONE },
-    { SCENE_KOEPONARACE, RESTRICTIONS_NO_DOUBLE_TIME },
-    { SCENE_GORONRACE, RESTRICTIONS_NONE },
-    { SCENE_TOWN, RESTRICTIONS_NONE },
-    { SCENE_ICHIBA, RESTRICTIONS_NONE },
-    { SCENE_BACKTOWN, RESTRICTIONS_NONE },
-    { SCENE_CLOCKTOWER, RESTRICTIONS_NONE },
-    { SCENE_ALLEY, RESTRICTIONS_NONE },
+#include "tables/scene_table.h"
```

Comment:
Very minor and nit picky and possibly something to do after all the tables are in, but it looks like enums have an indent for the include, while arrays don't (also happens in your other PRs and the other tables already included). Should we pick one style over the other? (personally I prefer the indent).

Reply:
I prefer the indent too, but formater removes it from the C files. Since we don't format the headers then we end up with that inconsistency

Reply:
Right, forgot about the formater. Never mind then :) .